### PR TITLE
Update Italian localization

### PR DESCRIPTION
**Pull request by M0Rf30**
As in the title,
Regards

### CODE REVIEW

**Comment by AJenbo** [comment]
Good timing

**Comment by AJenbo** [comment]
There's going to be and update to DevilutionX soon so now would be a good time to update the translation if you have time.

**Comment by M0Rf30** [comment]
A guy from devilutionX italian discord contacted me to know if it's possible to localize only the speeches, without involving itemDescriptions or itemNames or monsterNames, only dialogues between characters and things strictly related to the narration.

do you think it is feasible?

Why: Motivation is closely related to deep habits around the names of entities, reported here and there on wikis and communities.

TLDR
filter out entities localizations and only localize narration dialogues

**Comment by AJenbo** [comment]
Hm yeah I think that might be possible. I think it should be a general option not distinctly related to any given language.

Could you create an issue for it?
We should also have an option for not using the voice over translations in the same menu.
